app/web/src/components/ProjectsOfUserCell/ProjectsOfUserCell.tsx

```diff
@@ -8,6 +8,7 @@ export const QUERY = gql`
       id
       title
       mainImage
+      cadPackage
```

Comment:
@Irev-Dev I didn't catch that we didn't have the cadPackage badges showing on the UserProfile before!

Reply:
It wasn't related to `onClick` because I made that optional, but rather the reworking to make it a config with `dotClasses` and `buttonClasses` based on the current CAD Package which was undefined in `UserProfile` because of this missing query field.

Reply:
Ah nice.